.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+**What problems does this PR solve?**
```

Comment:
I guess this could be provided as a reference to an issue, perhaps that could be mentioned? I.e. something like "Provide a short description or reference to the relevant issue, explaining what problems this PR solves."

Reply:
Sure, I thought I'd keep it pretty general, since there isn't always a issue (esp. not one on Github) connected to a PR. I'll add your suggestion. :+1: